test/e2e/network/dns_common.go

```diff
@@ -280,6 +283,38 @@ func (t *dnsTestCommon) createDNSServer(aRecords map[string]string) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
+func (t *dnsTestCommon) createDNSServer(aRecords map[string]string) {
+	t.createDNSPodFromObj(generateDNSServerPod(aRecords))
+}
+
+func (t *dnsTestCommon) createDNSServerWithPtrRecord() {
+	pod := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "e2e-dns-configmap-dns-server-",
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:  "dns",
+					Image: imageutils.GetE2EImage(imageutils.DNSMasq),
+					Command: []string{
+						"/bin/sh", "-c",
+						"echo \"192.0.2.123 my.test\" > /etc/dnsmasq-hosts && " +
```

Comment:
Is it possible to make the test PTR record an argument of the function? Seems easy to break if we need to keep them (here and the actual test) consistent.

Reply:
Found something from the manual. Fixed.